src/main/kotlin/stringcalculator/InputConverter.kt

```diff
@@ -0,0 +1,32 @@
+package stringcalculator
+
+import stringcalculator.operations.Add
+import stringcalculator.operations.Divide
+import stringcalculator.operations.Multiply
+import stringcalculator.operations.Operation
+import stringcalculator.operations.Subtract
+
+class InputConverter {
+    fun extractInts(splitString: List<String>): ArrayDeque<Int> {
+        return ArrayDeque(splitString.filter { isNumeric(it) }.map { it.toInt() })
+    }
```

Comment:
```suggestion
    fun extractInts(splitString: List<String>): ArrayDeque<Int> = 
        ArrayDeque(splitString.filter { isNumeric(it) }.map { it.toInt() })
```
`=` 을 사용하면 return문을 생략할 수 있어요